build/dsc/manager.rb

```diff
@@ -188,6 +188,8 @@ def document_types(markdown_file_path, dsc_types)
           end
           file.write("\n")
         end
+        
+        file.write("\nGit reference of DSC used to create types: #{ENV['DSC_REF']}")        
```

Comment:
FYI - `DSC_REF` is not something that must be specified to perform a `dsc:build` ... so you might want to actually just do a `git rev-parse --short HEAD` against the import directory instead.